README.md

```diff
@@ -174,6 +174,11 @@ VARIABLE_WITH_DEFAULT: !var:default='defaultvalue' path/to/variable
     summon -D ENV=production --yaml 'SQL_PASSWORD: !var env/$ENV/db-password' deploy.sh
     ```
 
+* `--yaml` secrets.yml as a literal string.
```

Comment:
It might be clearer to add syntax to point out the expected literal YAML. Also, might be clearer to say that the literal YAML replaces `secrets.yml` file (it could be interpreted sorta reads as if the `secrets.yml` file could be getting passed here), something like:
```
`--yaml <YAML-string>` Passes secrets.yml as a literal string

Passes a literal YAML string to be parsed by the provider in place of a
`secrets.yml` file (see example above).
```

Also, Line 151 (not in your change set) could also similarly use a `<path-to-provider>` field (https://github.com/cyberark/summon/pull/198/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R151).

Same for Line 182 for `-i, --ignore` (https://github.com/cyberark/summon/pull/198/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R182)

Reply:
ok, Will add these to a follow-on commit.